client/src/components/PublicLobbyList.tsx

```diff
@@ -66,7 +64,11 @@ export function PublicLobbyList(props: PublicLobbyListProps) {
                     {`${lobby.state?.playerCount ?? 0}/${lobby.initialConfig.capacity}`}
                   </div>
                 </td>
-                <td className={"flex justify-center px-0.5 py-1 text-xs"}>
+                <td
+                  className={`col-span-2 flex justify-center px-0.5 py-1 text-xs border-r ${
+                    index % 2 === 0 ? "border-secondary-400" : "border-secondary-600"
```

Comment:
i just learned that tailwind has even and odd selectors https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even do you think we could use that?